org.freeplane.App.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Ismael Olea <ismael@olea.org> -->
+<component type="desktop">
+   <id>org.freeplane.App.desktop</id>
```

Comment:
```suggestion
   <id>org.freeplane.App</id>
```
and add a `<launchable type="desktop-id">org.freeplane.App.desktop</launchable>`